share/normalize/links.py

```diff
@@ -922,3 +930,10 @@ def __init__(self, func):
 
     def execute(self, obj):
         return list(filter(self._func, obj))
+
+
+class SourceIDLink(AbstractLink):
+    URN_FORMAT = 'urn:share:{source}:{id}'
+
+    def execute(self, obj):
+        return self.URN_FORMAT.format(source=Context()._config.label, id=obj)
```

Comment:
Should id be url encoded? For some of the ids I've just been sending back urls 😨